library/kani/src/arbitrary.rs

```diff
@@ -161,3 +161,12 @@ impl Arbitrary for std::marker::PhantomPinned {
         PhantomPinned
     }
 }
+
+impl<T> Arbitrary for std::boxed::Box<T>
+where
+    T: Arbitrary,
+{
+    fn any() -> Self {
+        Box::new(T::any())
```

Comment:
Does this cover all possible states a `Box` can have? What about a box created using `Box::new_uninit`?

Reply:
`Box::new_uninit` wouldn't fit this signature though, since T would have to be `MaybeUninit<U>`